### DISCUSSION TITLE
Access it at: [http://192.168.188.26:22900/], but 127.0.0.1 is still listening on 22900

### DISCUSSION THREAD

**Discussion by onurbi** [nzymedefense/nzyme#454] [Questions] [answered]
Hi,

nzyme is runnnig local on an raspi (5.10.17-v7l+) well. I want to run it headless but I found no way to open the port 22900 for the LAN IP.

I configured the http listening port 

`http_external_uri: "http://192.168.188.26:22900/"`

The log reports after starting

` INFO  horse.wtf.nzyme.NzymeLeaderImpl - Started web interface and REST API at [http://127.0.0.1:22900/]. Access it at: [http://192.168.188.26:22900/]`.

But netstat still reports only localhost is listening on 22900:

`$ netstat -an | grep 22900
tcp6       0      0 127.0.0.1:22900         :::*                    LISTEN
tcp6       0      0 127.0.0.1:22900         127.0.0.1:43136         TIME_WAIT
tcp6       0      0 127.0.0.1:22900         127.0.0.1:43482         VERBUNDEN
tcp6       0      0 127.0.0.1:22900         127.0.0.1:43332         VERBUNDEN
`
What could be the reason?

Onurbi

**Comment by lennartkoopmann** [comment, marked as answer]
You have to configure the `rest_listen_uri`, too: https://github.com/lennartkoopmann/nzyme/blob/master/nzyme.conf.example#L71

**Comment by onurbi** [comment]
Ok, both URIs has to be the same IP!

Now it works! Thanks!

I assumed, that the rest_listen_uri can stay at localhost.

In what case should they be configured differently?
